Document Company list settings configuration (#705)
mautic/user-documentation: docs/configuration/settings.rst

Company settings
****************

Company merge settings
======================

.. image:: images/company-merge-settings.png
  :width: 600
  :alt: Screenshot showing Company Merge Settings Configuration in Mautic

* **Merge by unique fields with operator** - You can determine which operator to use when merging fields if there is more than one unique identifier.

Company list settings
=====================

.. image:: images/company-list-settings.png
  :width: 600

>   :alt: Screenshot showing Company List Settings Configuration in Mautic

Reviewer: @promptless-for-oss there are a couple of things:

- Replace `images/company-list-settings.png` with the image below, and rename the file to use underscore, not dash.
- Fix the indentation here to follow our RST style guide.

```suggestion
.. image:: images/company_list_settings.png
   :width: 600
   :alt: Screenshot showing Company List Settings Configuration in Mautic
```

**Image:**

<img width="767" height="370" alt="Image" src="https://github.com/user-attachments/assets/305466da-fce1-4b3a-9821-d29e1dea9c15" />

> * **Columns** - Under **Configuration** and **Company Settings**, use this section to manage the visible columns for Companies. Move fields from the left list to the right list to display them on the main Companies overview, or remove fields from the right list to hide them from the view.

Reviewer: ```suggestion
* **Columns** - Manage the visible columns for Companies in this section. Move fields from the left list to the right list to display them on the main Companies overview, or remove fields from the right list to hide them.
```

Company settings
****************

Company merge settings
======================

.. image:: images/company-merge-settings.png
  :width: 600
  :alt: Screenshot showing Company Merge Settings Configuration in Mautic

* **Merge by unique fields with operator** - You can determine which operator to use when merging fields if there is more than one unique identifier.

> Company list settings

Author: New Company List Settings feature added in [PR #16072](https://github.com/mautic/mautic/pull/16072). The PR adds a `company_columns` configuration in `ConfigCompanyType.php` and `CompanyColumnsDictionary.php` service to allow users to customize which columns display in the Company list view, mirroring the existing Contact list settings functionality.

Source: https://github.com/mautic/mautic/pull/16072

=====================

.. image:: images/company-list-settings.png
  :width: 600
  :alt: Screenshot showing Company List Settings Configuration in Mautic

* **Columns** - Select from the left which fields appear on the Company list (when you go to Companies in Mautic and view the list).

> To display the fields, select them from the left and move them to the right column, or remove from the right column if you don't want them to appear in the list.

Reviewer: ```suggestion
* **Columns** - Under **Configuration** and **Company Settings**, use this section to manage the visible columns for Companies. Move fields from the left list to the right list to display them on the main Companies overview, or remove fields from the right list to hide them from the view.
```

Company settings
****************

Company merge settings
======================

.. image:: images/company-merge-settings.png
  :width: 600
  :alt: Screenshot showing Company Merge Settings Configuration in Mautic

* **Merge by unique fields with operator** - You can determine which operator to use when merging fields if there is more than one unique identifier.

Company list settings
=====================

.. image:: images/company_list_settings.png
   :width: 600

>    :alt: Screenshot showing Company List Settings Configuration in Mautic

Reviewer: Reword the alt text to be descriptive and don't use word "secreenshot".

> * **Columns** - Manage the visible columns for Companies in this section. Move fields from the left list to the right list to display them on the main Companies overview, or remove fields from the right list to hide them.

Reviewer: ```suggestion
* **Columns** - Manage the visible columns for Companies. Move fields from the left list to the right list to display them on the main Companies overview, or remove fields from the right list to hide them.
```
